app/controllers/item.js

```diff
@@ -46,8 +47,10 @@ export default Ember.Controller.extend({
   linkDisplayName: Ember.computed("prevPath", function() {
     let prevPath = this.get("prevPath");
     if (prevPath === "search_goods") {
+      this.set("showPrevNextButtons", false);
```

Comment:
we can't `set` anything in a computed getter. It is likely to cause unpredictable side-effects in the future. And we cannot guarantee that the computed property will ever be computed at the right time (if ever).

Reply:
maybe this instead: 

```javascript
showPrevNextButtons: Ember.computed("prevPath", function() {
   return prevPath !== "search_goods"
});
```

Reply:
okay, looks better. 
Thanks :)